rnaget-openapi.yaml

```diff
@@ -887,3 +1314,82 @@ components:
       properties:
         message:
           type: string
+  parameters:
+    versionParam:
+      name: version
+      in: query
+      description: version to filter by
+      required: false
+      schema:
+        type: string
+      example:
+        1.0
+    studyIDParam:
+      name: studyID
+      in: query
+      description: study to filter by
+      required: false
+      schema:
+        type: string
+      example:
+        c4cf910c9ae54832902c954cb439e30c
+    sampleIDListParam:
+      name: sampleIDList
+      in: query
+      description: comma separated list of sampleIDs to match
+      required: false
+      style: form
+      explode: false
+      schema:
+        type: array
```

Comment:
I am wondering if `sampleIDList`, `featureIDList`, and `featureNameList` are `arrays`, or if they are actually `strings`.

To illustrate, this is how `sampleIDList` is currently submitted (i.e. a `string`):
?sampleIDList=GENE1,GENE2,GENE3,GENE4

If `sampleIDList` was an `array`, it would be submitted like this:
?sampleIDList=["GENE1","GENE2","GENE3","GENE4"]

Thoughts?

Reply:
We should follow what is in the OpenAPI and have it be an `array`.  References to comma-separated lists (i.e. `string`) are no longer in the spec.